src/ralph/settings/base.py

```diff
@@ -124,7 +124,7 @@ def os_env_true(var, default=''):
 TIME_ZONE = 'Europe/Warsaw'
```

Comment:
This would entail changes to test fixtures that use Warsaw timezones. Should they be changed to use dynamic timezone from the environment?